RELEASES.md

```diff
@@ -1,3 +1,45 @@
+## 1.16
+
+### For iOS and Android
+
+### Features
+
+- EIP-1159 support ([details](https://github.com/status-im/status-react/pull/12369))
+- Hide/show option for accounts in the wallet ([details](https://github.com/status-im/status-react/pull/12438))
+- Delete accounts in wallet ([details](https://github.com/status-im/status-react/pull/12444))
+- Enable keeping database when migrating account to keycard ([details](https://github.com/status-im/status-react/pull/12306))
+- Replies from group chat in activity centre ([details](https://github.com/status-im/status-react/pull/12266))
+- Enable transaction management (alpha, can be enabled in "Advanced") ([details](https://github.com/status-im/status-react/pull/12407))
```

Comment:
Let's remove it, as mostly it is not working - so no reason to promote it

Reply:
I meant "- Enable transaction management (alpha, can be enabled in "Advanced") ([details](https://github.com/status-im/status-react/pull/12407))"

Reply:
Done